test/box-luatest/gh_6403_space_pairs_luafun_integration_test.lua

```diff
@@ -0,0 +1,44 @@
+local server = require('luatest.server')
+local t = require('luatest')
+
+-- Both engines to check if both luac and ffi implementations work correctly
+local g = t.group(nil, {{engine = 'memtx'}, {engine = 'vinyl'}})
+
+g.before_all(function(g)
+    g.server = server:new()
+    g.server:start()
+    g.server:exec(function(engine)
+        local space = box.schema.space.create('test', {engine = engine})
+        space:create_index('primary')
+        space:insert({1})
+        space:insert({2})
+        space:insert({3})
+    end, {g.params.engine})
+end)
+
+g.after_all(function(g)
+    g.server:drop()
+end)
+
+-- Check if drop_while method works fine for space.pairs
```

Comment:
Nit: missed dot at the end of the sentence.

Reply:
Fixed.